<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ASCDK041" name="Test" tests="1" failures="1" timestamp="15 Apr 2016 10:11:13 GMT" time="62.711" errors="0">
  <testcase name="rechargePayTM" time="34.523" classname="com.pack.test.RechargeTest">
    <failure type="java.lang.AssertionError" message="expected [Airtel] but found [AirteL]">
      <![CDATA[java.lang.AssertionError: expected [Airtel] but found [AirteL]
	at com.pack.page.Recharge.VerifyOperatorName(Recharge.java:63)
	at com.pack.test.RechargeTest.rechargePayTM(RechargeTest.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- rechargePayTM -->
</testsuite> <!-- Test -->
